dags/pytorch_xla/r2_5.py

```diff
@@ -186,6 +208,14 @@ def llama():
       ),
       US_CENTRAL2_B_TPU_PROD_ENV,
   )
+  llama_3_train_v5p_8 = task.run_queued_resource_test(
+      test_config.JSonnetTpuVmTest.from_pytorch(
+          "pt-2-5-llama3-train-func-v5p-8-1vm",
```

Comment:
We need the `reserved=True` flag for the v5p resource. I spent long time to figure this out :<

Reply:
Thanks, added

Reply:
do we need this on llama2 v5p too?, i just added for llama2 on v5p directly too

Reply:
thanks, added